user_guide_src/source/libraries/caching.rst

```diff
@@ -67,7 +67,7 @@ This is an array of settings specific to the  ``File`` handler to determine how
 $memcached
 ==========
 
-This is an array of servers that will be used when using the ``Memcache(d)`` handler.
+This is an array of servers that will be used when using the ``Memcached`` handler.
```

Comment:
The classname is `MemcachedHandler`. So it should be like the following?
```
the ``MemcachedHandler`` handler
```
or just
```
``MemcachedHandler``
```

Reply:
@kenjis Then should I change these lines too?
_This is commonly the **File handler** since the file system is always available, but may not fit more complex, multi-server setups._
_This is an array of settings specific to the **File handler** to determine how it should save the cache files._
_This is an array of servers that will be used when using the **Memcache(d) handler**._
_The settings for the Redis server that you wish to use when using the **Redis and Predis handler**._
Or just fix the typo, remove the parentheses, **Memcache(d)** to **Memcached**?

Reply:
If it indicates the classname, decorate with ` `` `, but it is not, we don't use ` `` ` decoration.
In these cases, the classnames have `Handler` suffix. So the current notations are not good.

1. use the classnames and ` `` `
```
This is commonly the ``FileHandler`` handler ...
```
2. do not use ` `` `
```
This is commonly the File handler ...
```
or
```
This is commonly the **File** handler ...
```

Reply:
I think the parentheses in "Memcache(d)" should be removed.